### ISSUE TITLE
lipid and enzyme terms

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#4214]
Preferred term label (e.g., asplenia)

1. increased circulating chylomicron triglyceride level
2. decreased circulating chylomicron triglyceride level
3. abnormal malondialdehyde level
4. increased circulating malondialdehyde level
5. decreased circulating malondialdehyde level 
6. increased liver malondialdehyde level
7. decreased liver malondialdehyde level
8. increased liver superoxide dismutase level
9. decreased liver superoxide dismutase level
10. increased kidney superoxide dismutase level
11. decreased kidney superoxide dismutase level
12. increased liver catalase activity 
13. decreased liver catalase activity 
14. increased circulating glutathione reductase activity
15. decreased circulating glutathione reductase activity
16. increased circulating glutathione peroxidase activity
17. decreased circulating glutathione peroxidase activity
18. increased liver glutathione peroxidase activity
19. decreased liver glutathione peroxidase activity
20. abnormal circulating CCL20 level
21. increased circulating CCL20 level
22. decreased circulating CCL20 level
23. abnormal circulating CSF2 level
24. increased circulating CSF2 level 
25. decreased circulating CSF2 level
26. abnormal circulating CCL2 level
27. increased circulating CCL2 level
28. decreased circulating CCL2 level

Synonyms (e.g., absent spleen)

1. increased circulating ULDL triglyceride level; increased circulating ultra-low-density lipoprotein
2. decreased circulating ULDL triglyceride level; increased circulating ultra-low-density lipoprotein
3. abnormal MDA level
4. increased circulating MDA level
5. decreased circulating MDA level 
6. increased liver MDA level
7. decreased liver MDA level
8. increased liver SOD level
9. decreased liver SOD level
10. increased kidney SOD level
11. decreased kidney SOD level
12. increased liver CAT activity 
13. decreased liver CAT activity 
14. increased circulating GR activity; increased circulating glutathione-disulfide reductase activity; decreased circulating GSR activiin
15. decreased circulating GR activity; decreased circulating glutathione-disulfide reductase activity; decreased circulating GSR activity
16. increased circulating GSH-Px activity; increased circulating GSHPx activity; increased circulating GPX activity
17. decreased circulating GSH-Px activity; decreased circulating GSHPx activity; decreased circulating GPX activity
18. increased liver GSH-Px activity; increased liver GSHPx activity; increased liver GPX activity
19. decreased liver GSH-Px activity; decreased liver GSHPx activity; increased liver GPX activity
20. abnormal circulating C-C motif chemokine ligand 20 level; abnormal circulating MIP3A level; abnormal circulating macrophage inflammatory protein 3 alpha level
21. increased circulating C-C motif chemokine ligand 20 level; increased circulating MIP3A level; increased circulating macrophage inflammatory protein 3 alpha level
22. decreased circulating C-C motif chemokine ligand 20 level; decreased circulating MIP3A level; decreased circulating macrophage inflammatory protein 3 alpha level
23. abnormal circulating colony stimulating factor 2 level; abnormal circulating granulocyte-macrophage colony-stimulating factor level; abnormal circulating GM-CSF level
24. increased circulating colony stimulating factor 2 level; increased circulating granulocyte-macrophage colony-stimulating factor level; increased circulating GM-CSF level
25. decreased circulating colony stimulating factor 2 level; decreased circulating granulocyte-macrophage colony-stimulating factor level; decreased circulating GM-CSF level
26. abnormal circulating C-C motif chemokine ligand 2 level; abnormal circulating monocyte chemotactic protein 1 level; abnormal circulating MCP1 level
27. increased circulating C-C motif chemokine ligand 2 level; increased circulating monocyte chemotactic protein 1 level; increased circulating MCP1 level
28. decreased circulating C-C motif chemokine ligand 2 level; decreased circulating monocyte chemotactic protein 1 level; decreased circulating MCP1 level

Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

1. increased circulating triglyceride level (MP:0001552)
2. decreased circulating triglyceride level (MP:0002644)
3. abnormal homeostasis (MP:0001764)
4. abnormal malondialdehyde level (new term)
5. abnormal malondialdehyde level (new term)
6. abnormal malondialdehyde level (new term)
7. abnormal malondialdehyde level (new term)
8. increased superoxide dismutase level (MP:0012660)
9. decreased superoxide dismutase level (MP:0012659)
10. increased superoxide dismutase level (MP:0012660)
11. decreased superoxide dismutase level (MP:0012659)
12. increased catalase activity (MP:0011593)
13. decreased catalase activity (MP:0011594)
14. increased glutathione reductase level (MP:0012630)
15. decreased glutathione reductase level (MP:0012629)
16. increased glutathione peroxidase activity (MP:0011602)
17. decreased glutathione peroxidase activity (MP:0011603)
18. increased glutathione peroxidase activity (MP:0011602)
19. decreased glutathione peroxidase activity (MP:0011603)
20. abnormal circulating chemokine level (MP:0010209)
21. abnormal circulating CCL20 level (new term)
22. abnormal circulating CCL20 level (new term)
23. abnormal circulating cytokine level (MP:0010210)
24. abnormal circulating CSF2 level (new term)
25. abnormal circulating CSF2 level (new term)
26. abnormal circulating chemokine level (MP:0010209)
27. abnormal circulating CCL2 level (new term)
28. abnormal circulating CCL2 level (new term)

Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

1. increase in the amount of chylomicrons, a class of lipoproteins (approximate density range <0.95 g/ml) that transport exogenous triglycerides and cholesterol from the small intestine to other tissues, carrying triglyceride in the blood, plasma, or serum
2. decrease in the amount of chylomicrons, a class of lipoproteins (approximate density range <0.95 g/ml) that transport exogenous triglycerides and cholesterol from the small intestine to other tissues, carrying triglyceride in the blood, plasma, or serum
3. aberrant concentration of malondialdehyde, an organic compound with the formula CH2(CHO)2. Malondialdehyde is a reactive species which occurs naturally, is generally found in the enol form (HOCH=CH-CHO), and is a marker for oxidative stress.
4. increase in the amount of circulating malondialdehyde, an organic compound with the formula CH2(CHO)2. Malondialdehyde is a reactive species which occurs naturally, is generally found in the enol form (HOCH=CH-CHO), and is a marker for oxidative stress.
5. decrease in the amount of circulating malondialdehyde, an organic compound with the formula CH2(CHO)2. Malondialdehyde is a reactive species which occurs naturally, is generally found in the enol form (HOCH=CH-CHO), and is a marker for oxidative stress.
6. increase in the amount of liver malondialdehyde, an organic compound with the formula CH2(CHO)2. Malondialdehyde is a reactive species which occurs naturally, is generally found in the enol form (HOCH=CH-CHO), and is a marker for oxidative stress.
7. decrease in the amount of liver malondialdehyde, an organic compound with the formula CH2(CHO)2. Malondialdehyde is a reactive species which occurs naturally, is generally found in the enol form (HOCH=CH-CHO), and is a marker for oxidative stress.
8. increase in the amount of the liver oxidoreductase that catalyzes the reaction between superoxides and hydrogen to yield molecular oxygen and hydrogen peroxide
9. decrease in the amount of the liver oxidoreductase that catalyzes the reaction between superoxides and hydrogen to yield molecular oxygen and hydrogen peroxide
10. increase in the amount of the kidney oxidoreductase that catalyzes the reaction between superoxides and hydrogen to yield molecular oxygen and hydrogen peroxide
11. decrease in the amount of the kidney oxidoreductase that catalyzes the reaction between superoxides and hydrogen to yield molecular oxygen and hydrogen peroxide
12. greater capacity to catalyze the reaction: 2 hydrogen peroxide = O2 + 2 H2O, in the liver
13. lower capacity to catalyze the reaction: 2 hydrogen peroxide = O2 + 2 H2O, in the liver
14. increase in the amount of the enzyme that catalyzes the oxidation of glutathione to glutathione disulfide in the presence of NADP+ in the blood
15. decrease in the amount of the enzyme that catalyzes the oxidation of glutathione to glutathione disulfide in the presence of NADP+ in the blood
16. increased ability to catalyze the reaction: 2 glutathione + hydrogen peroxide = oxidized glutathione + 2 H2O in blood
17. decreased ability to catalyze the reaction: 2 glutathione + hydrogen peroxide = oxidized glutathione + 2 H2O in blood
18. increased ability to catalyze the reaction: 2 glutathione + hydrogen peroxide = oxidized glutathione + 2 H2O in liver
19. decreased ability to catalyze the reaction: 2 glutathione + hydrogen peroxide = oxidized glutathione + 2 H2O in liver
20. deviation from the normal blood levels of CCL20 protein,  a chemokine that binds the CCR6 chemokine receptor, and is involved in cellular response to bacterial infection.
21. an increase in the normal blood levels of CCL20 protein,  a chemokine that binds the CCR6 chemokine receptor, and is involved in cellular response to bacterial infection.
22. a decrease in the normal blood levels of CCL20 protein,  a chemokine that binds the CCR6 chemokine receptor, and is involved in cellular response to bacterial infection.
23. deviation from the normal blood levels of CSF2, a cytokine that controls the production, differentiation, and function of granulocytes and macrophages. 
24. increase in the the normal blood levels of CSF2, a cytokine that controls the production, differentiation, and function of granulocytes and macrophages. 
25. decrease in the the normal blood levels of CSF2, a cytokine that controls the production, differentiation, and function of granulocytes and macrophages. 
26. deviation from the normal blood levels of CCL2 protein,  a chemokine that is involved in leukocyte chemotaxis and migration.
27.  increase in the the normal blood levels of CCL2 protein,  a chemokine that is involved in leukocyte chemotaxis and migration.
28. decrease in the the normal blood levels of CCL2 protein,  a chemokine that is involved in leukocyte chemotaxis and migration.

PMID:27871290

If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

ORCID:0000-0001-5356-4174

**Comment by sbello** [comment]
gotten to request 11
I've been adding the abnormal parent term in each case
also added muscle for the SOD terms as MGI has some annotations for this
